smartsim/experiment.py

```diff
@@ -868,49 +868,48 @@ def reconnect_orchestrator(self, checkpoint: str) -> Orchestrator:
     def preview(
         self,
         *args: t.Any,
-        output_format: previewrenderer.Format = previewrenderer.Format.PLAINTEXT,
         verbosity_level: previewrenderer.Verbosity = previewrenderer.Verbosity.INFO,
+        output_format: previewrenderer.Format = previewrenderer.Format.PLAINTEXT,
         output_filename: t.Optional[str] = None,
     ) -> None:
         """Preview entity information prior to launch. This method
         aggregates multiple pieces of information to give users insight
         into what and how entities will be launched.  Any instance of
         ``Model``, ``Ensemble``, or ``Orchestrator`` created by the
         Experiment can be passed as an argument to the preview method.
-        :param output_filename: Specify name of file and extension to write
-        preview data to. If no output filename is set, the preview will be
-        output to stdout. Defaults to None.
-        :type output_filename: str
-        :param output_format: Set output format. The possible accepted
-        output formats are `json`, `xml`, `html`, `plain_text`, `color_text`.
-        Defaults to 'plain_text'.
-        :type output_type: str
-        :param verbosity_level: Specify the verbosity level:
-            info: Display User defined fields and entities
-            debug: Display user defined field and entities and auto generated
+
+        Verbosity levels:
+            - info: Display user-defined fields and entities
+            - debug: Display user-defined field and entities and auto-generated
             fields.
-            developer: Display user defined field and entities, auto generated
+            - developer: Display user-defined field and entities, auto-generated
             fields, and run commands.
-            Defaults to info.
+
+        :param verbosity_level: verbosity level specified by user, defaults to info.
         :type verbosity_level: str
+        :param output_format: Set output format. The possible accepted
+            output formats are `json`, `xml`, `html`, `plain_text`, `color_text`.
+            Defaults to 'plain_text'.
+        :type output_type: str
```

Comment:
I think this is supposed to be `:type output_format:`

Reply:
good catch, changed.